### ISSUE TITLE
setting current_tenant in spec_helper throws error

### ISSUE THREAD

**Issue by thorion3006** [ErwinM/acts_as_tenant#218]
As per the documentation I set my current_tenant in the spec_helper as:
```
config.before(:suite) do
  $default_company = FactoryBot.create(:company)
end

config.before(:each) do
  if example.metadata[:type] == :request
    ActsAsTenant.test_tenant = $default_company
  else
    ActsAsTenant.current_tenant = $default_company
  end
end

config.after(:each) do
  ActsAsTenant.current_tenant = nil
  ActsAsTenant.test_tenant = nil
end
```

but when I run RSpec I get the following errors:
```
1.1) Failure/Error: if example.metadata[:type] == :request
        `example` is not available from within an example (e.g. an `it` block) or from constructs that run in the scope of an example (e.g. `before`, `let`, etc). It is only available on an example group (e.g. a `describe` or `context` block).
     # ./spec/spec_helper.rb:45:in `block (2 levels) in <top (required)>'

1.2) Failure/Error: ActsAsTenant.test_tenant = nil
      
     NoMethodError:
       undefined method `test_tenant=' for ActsAsTenant:Module
     # ./spec/spec_helper.rb:58:in `block (2 levels) in <top (required)>'
```

I am running:
RSpec 3.9, Rails 6.0.1

**Comment by ErwinM** [comment]
You can probably fix this like so #215 

I will be away traveling the next 2 weeks. When I get back I will release the latest version to RubyGems (not a good idea to do so, when you do not have access to your pc..).

**Comment by joshukraine** [comment]
@ErwinM Adding AaT to a new project this week and ran into this problem. Currently using the GitHub workaround as suggested in #215. Any chance we could get a RubyGems release soon? 😬🙏🏻

Thanks for your work on maintaining this gem! 🙂
